templates/swagger-ui/swagger-ui-onload.js

```diff
@@ -0,0 +1,19 @@
+window.onload = function() {
+    const ui = SwaggerUIBundle({
+        url: "static/swagger.json",
+        dom_id: '#swagger-ui',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Seagate/cortx-manager/pullRequest?prid=9452599)
```suggestion
        dom_id: "#swagger-ui",
```